django_nose/runner.py

```diff
@@ -381,7 +381,8 @@ def _foreign_key_ignoring_handle(self, *fixture_labels, **options):
             connection.close()
 
 
-def _skip_create_test_db(self, verbosity=1, autoclobber=False, serialize=True):
+def _skip_create_test_db(self, verbosity=1, autoclobber=False, serialize=True,
+                         keepdb=True):
```

Comment:
This keyword arg is passed now by Django 1.8, so it needs to be added here to prevent the following error:

```
File "/[directory_path]/lib/python2.7/site-packages/django/test/runner.py", line 370, in setup_databases
    serialize=connection.settings_dict.get("TEST", {}).get("SERIALIZE", True),
TypeError: _skip_create_test_db() got an unexpected keyword argument 'keepdb'
```